### DISCUSSION TITLE
Github Student Pack , cant use Azure?

### DISCUSSION THREAD

**Discussion by Lanson2232** [community/community#23142] [New to GitHub]
<p>I’m a member of Student pack and i want use Microsoft Azure credits But i dont have a School mail To use Azure student or a code ? if i can get a code for verify , where can i find it?<br>
(English is not my native language, Sry)</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/lanson2232">@lanson2232</a>,</p>
<p>Thank you for being here! Unfortunately, we have recently the removed the Azure benefit from the GitHub Student Developer Pack for now. It should return in the future.</p>
<p>Scratch that! :slight_smile: You should be able to use it now, If you have questions or issues with an offer, please contact Microsoft directly for assistance at this link: <a href="https://support.microsoft.com/supportrequestform/7a3f017a-c951-7e06-1bd7-64bf7fc202ff" rel="nofollow noopener">https://support.microsoft.com/supportrequestform/7a3f017a-c951-7e06-1bd7-64bf7fc202ff</a></p>

**Comment by pallavi7mentor** [comment]
<p>Azure for Students is available only to students who meet the following requirements. You must affirm that you are age 18 or older and attend an accredited, degree-granting two-year or four-year educational institution where you’re a full-time student in a STEM-related field. You must verify your academic status through your organisation’s email address . This offer is not available for use in a massive open online course (MOOC) or in other professional trainings from for-profit organisations. The <a href="https://www.sevenmentor.com/azure-training-institute-in-pune.php" rel="nofollow noopener">Azure</a> for Students offer is limited to one per eligible student, is non-transferable and cannot be combined with any other offers unless otherwise permitted by Microsoft. Azure for Students has a limited supply of activations and the offer is available only while supplies last. Each region has a limited number of Azure for Students offers available and your geographical location may run out of supplies while the offer is still available in other regions. Microsoft may terminate this offer at our sole discretion. Other restrictions may apply.</p>

**Comment by burbigo1** [comment]
<p>Thank you so much, I didn’t know that this was possible</p>

**Comment by aaemon** [comment]
<p>It did not work. I contacted them twice first, one said they can not help with verification code and another redirected me to GitHub support. Github support redirected me to azure support again, they just ask me to login azure using sign in with GitHub, but it always redirects me to the verification page.</p>

**Comment by sudhanshu-maurya** [comment]
<p>I don’t have the school email id ,but i have access to github student pack then how can use Azure from that . please help me!!</p>

**Comment by un-ro** [comment]
<p>What ? removed ?<br>
Wow even some CS doesn’t know if azure is not available in Student.</p>
<p>I was contact Azure Support then say contact Github Support.<br>
Then from Github Support say I must contact Azure Support.</p>
<p>HHHHHHHHHHHHHHHHHHHHHHHHHHHHHHH</p>

**Comment by Shofikul-Isl4m** [comment]
even in 2025
